tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -86,35 +90,66 @@ class ConfigurationPassImpl @Inject()(val options: WriteablePassOptions,
         configFilename
     val basename = FilenameUtils.getName(filename)
     logger.info("  > Loading TLC configuration from " + basename)
+
+    def setInit(init: String): Unit = {
+      options.get[String]("checker","init") match {
+        case None =>
+          // In general, passes should not override options. This is a reasonable exception to this rule.
+          logger.info(s"  > Using the init predicate $init from the TLC config")
+          options.set("checker.init", init)
+
+        case Some(cmdInit) =>
+          logger.warn(s"  > $basename: Init operator is set both in TLC config and via --init option; using $cmdInit")
```

Comment:
To make it immediately clear which is taking precedence, and to be consistent with the messaging for `--next`:

```suggestion
          logger.warn(s"  > $basename: Init operator is set in TLC config but overridden via --init command line option; using $cmdInit")
```